main.py

```diff
@@ -135,6 +136,51 @@ async def handle_augments(screenshot: ndarray, adb_instance: ADB):
     await asyncio.sleep(1)
 
 
+async def check_surrender_state(adb_instance: ADB, screenshot: ndarray, config: AluneConfig):
```

Comment:
This is missing the return type and documentation about what it returns.

Reply:
Return type means `) -> bool:`

Reply:
mb